nuxeo-compound-documents/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -3,3 +3,4 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Nuxeo Compound Documents
 Bundle-SymbolicName: org.nuxeo.compound.documents;singleton:=true
 Bundle-Vendor: Nuxeo
+Nuxeo-Component: OSGI-INF/compound-documents-contrib.xml
```

Comment:
I would have renamed the file to something more specific about what it does, such as `core-types-contrib.xml` (like in other packages).
WDYT @kevinleturc ?